packages/nextjs/components/op/modals/VoteModal.tsx

```diff
@@ -78,6 +78,8 @@ const VoteModal: React.FC<IVoteModal> = ({
             onChange={e => handleAllocationChange(e.target.value)}
             className="border-none outline-none p-3 w-full  "
             value={allocation}
+            type="number"
```

Comment:
Changing this input to a number type makes it a worse UX. We should avoid using the number type for inputs.

Reply:
Why so??

Reply:
yeah, got it!